lib/signs_ui_web/templates/signs/index.html.eex

```diff
@@ -2,19 +2,26 @@
 <div class="container signs-container">
   <%= form_for @conn, signs_path(@conn, :update), [as: :signs, class: "signs-form"], fn f -> %>
     <div class="row signs-headers-row">
-      <div class="col-md-1 col-xs-2 sign-header">
+      <div class="col-xs-1 sign-header">
         <b class="pull-right">Status</b>
       </div>
-      <div class="col-md-11 col-xs-10">
+      <div class="col-xs-11">
         <b>Sign</b>
       </div>
     </div>
-    <%= for {id, label} <- @sign_names do %>
-      <%= if @signs[id] do %>
-          <%= sign_checkbox(id, @signs[id].enabled?, f) %>
-          <%= sign_label(id, label, @signs[id].enabled?) %>
+    <div class="row">
+      <%= for {id, label} <- @sign_names do %>
+        <%= if @signs[id] do %>
+            <%= sign_checkbox(id, @signs[id].enabled?, f) %>
+            <%= sign_label(id, label, @signs[id].enabled?) %>
+        <% end %>
       <% end %>
-    <% end %>
-    <%= submit "Submit", class: "btn btn-primary" %>
+    </div>
+    <div class="row">
+      <div class="col-xs-12">Don't see the sign you're looking for?</div>
```

Comment:
These were left-aligned with the switches in the mockups. Maybe remove the left padding on them? Also, maybe a little bit of vertical padding between the text and the submit button.